run

```diff
@@ -3,15 +3,15 @@
 set -eo pipefail
 
 function start {
-  docker-compose up --build -d "${@}"
+  docker compose up --build -d "${@}"
```

Comment:
@daneryl could you take a look at this? It looks like it's fine to change cause compose as part of docker cli is the next version.

Reply:
We agreed this is a good change. Resolving this comment.